lib/createlocaltracks.js

```diff
@@ -102,6 +102,11 @@ function createLocalTracks(options) {
       : {}
   };
 
+  // eslint-disable-next-line no-undefined
+  if (options.audio && options.audio.workaroundWebKitBug1208516 !== undefined) {
+    localTrackNameOptions.audio.workaroundWebKitBug1208516 = options.audio.workaroundWebKitBug1208516;
+  }
+
```

Comment:
@manjeshbhargav, I think this is needed because we although we do pass localTrackOptions down https://github.com/twilio/twilio-video.js/blob/master/lib/createlocaltracks.js#L130 - They are in the wrong format. LocalAudioTrack constructor expects options to be `{ workaroundWebKitBug1208516: true }`
But here they are passed as: `{  audio: { workaroundWebKitBug1208516: true  } }` 
I think the way we pass it down is a bug.